pkg/controller/sparkapplication/controller.go

```diff
@@ -650,6 +652,15 @@ func (c *Controller) submitSparkApplication(app *v1beta1.SparkApplication) *v1be
 		return app
 	}
 
+	// Use batch scheduler to perform app submit tasks
+	if c.scheduleViaBatchScheduler(app) {
+		var err error
+		app, err = c.batchScheduler.OnSubmitSparkApplication(app)
+		if err != nil {
+			glog.Errorf("Failed to process batch scheduler OnSubmitSparkApplication with error %v", err)
```

Comment:
Let's change the error message to `failed to apply batch scheduling to SparkApplication %s: %v`.